pkg/server/container_start.go

```diff
@@ -266,7 +265,7 @@ func (c *criContainerdService) generateContainerSpec(id string, sandboxPid uint3
 	// TODO(random-liu): [P0] Handle privileged.
 
 	// Set namespaces, share namespace with sandbox container.
-	setOCINamespaces(&g, securityContext.GetNamespaceOptions(), sandboxPid)
+	setOCINamespaces(&g, securityContext.GetNamespaceOptions(), sb)
```

Comment:
Do not pass in the SandboxMetadata. I want to make these functions `cri-containerd` independent, so that in the future we may be able to consolidate the logic with cri-o.